README.md

```diff
@@ -156,23 +172,41 @@ The Azure MCP Server provides Azure SDK and Azure CLI developer tools. It can be
 
 Here's a short (16 seconds) video to help you get the Azure MCP Server installed in VS Code.
 
-https://github.com/user-attachments/assets/535f393c-0ed2-479d-9b24-5ca933293c92
+<https://github.com/user-attachments/assets/535f393c-0ed2-479d-9b24-5ca933293c92>
 
 #### ✨ Quick install with VS Code
 
-Click one of these buttons to install the Azure MCP Server for VS Code or VS Code Insiders.
-
-[![Install with NPX in VS Code](https://img.shields.io/badge/VS_Code-Install_Azure_MCP_Server-0098FF?style=flat-square&logo=visualstudiocode&logoColor=white)](https://insiders.vscode.dev/redirect/mcp/install?name=Azure%20MCP%20Server&config=%7B%22command%22%3A%22npx%22%2C%22args%22%3A%5B%22-y%22%2C%22%40azure%2Fmcp%40latest%22%2C%22server%22%2C%22start%22%5D%7D) [![Install with NPX in VS Code Insiders](https://img.shields.io/badge/VS_Code_Insiders-Install_Azure_MCP_Server-24bfa5?style=flat-square&logo=visualstudiocode&logoColor=white)](https://insiders.vscode.dev/redirect/mcp/install?name=Azure%20MCP%20Server&config=%7B%22command%22%3A%22npx%22%2C%22args%22%3A%5B%22-y%22%2C%22%40azure%2Fmcp%40latest%22%2C%22server%22%2C%22start%22%5D%7D&quality=insiders)
+Below is a list of available Azure MCP servers. Click either install button to quickly set up a specific service in VS Code or VS Code Insiders:
 
 You might get prompted to open the link with VS Code and then select "Install Server".
 
-Once you've installed the Azure MCP Server, make sure you select GitHub Copilot Agent Mode and refresh the tools list. To learn more about Agent Mode, visit the [VS Code Documentation](https://code.visualstudio.com/docs/copilot/chat/chat-agent-mode).
+Once you've installed the Azure MCP Server(s), make sure you select GitHub Copilot Agent Mode and refresh the tools list. To learn more about Agent Mode, visit the [VS Code Documentation](https://code.visualstudio.com/docs/copilot/chat/chat-agent-mode).
```

Comment:
Please show how to install each one.  Or at least one and how the service name "bestpractices" etc